### DISCUSSION TITLE
Akash Zealy Snz 2 wrap up and Phase 3.0 Proposal intro

### DISCUSSION THREAD

**Discussion by ghost** [akash-network/community#728] [Governance Proposals]
### Introduction
The Akash Developer Onboarding Program on Zealy campaign is an incentive program designed to encourage direct contributions to the Akash network and enhance education about the Akash Network. 

The Developer Onboarding Program on Zealy Szn 2 has officially ended, in this discussion we'll share how Szn 2 went and introduce our proposal for Phase 3.0 to the Akash community.

### Expected Vs Actual outcomes

**Users**
Expected outcome 500-1000 unique users partaking in missions learning and earning from the different missions. 
Actual outcome:
- 726 Total Members in Zealy  
- 582 total participants during the 3 sprints of Zealy Season 2, a 400% increased compared to Season 1

**Budget** 
Expected outcome: $12,000 USD worth of role payments, $9,000 USD worth of rewards payments, and no additional spending. 
Actual outcome: $10,000 USD worth of role payments, $8,400 USD worth of rewards payments, plus $750 for Zealy subscription, plus $300 paid in extra rewards for the top 5 best of the best submissions. This leaves us with $2,700 in the Zealy wallet. *These are estimates as volatility was very present throughout the campaign 

### Best of the best submissions and 3 month recap doc:

Refer to this thread on X (formerly Twitter) to find the top 5 best of the best submissions found in the Dev Onboarding Program

https://x.com/instafinanzas/status/1856706657746645367

Refer to this document for resources, more specific details on what went good, what can be improved and future considerations for the Dev Onboarding Campaign on Zealy

[Akash Zealy Szn 2 Retrospective 3 month report ](https://docs.google.com/document/d/1boCBi6BWB6TPRTndzcEBvsadID5UQSV46pRuGP0LzsY/edit?usp=sharing)

### What is next? Enter Phase 3.0 proposal
The Akash Zealy campaign is an incentive program designed to encourage direct contributions to the Akash network and enhance education about the platform. It focuses on incentivizing developers and contributors through structured missions and rewards, increasing engagement from core and new contributors, and maintaining regular community involvement through meetings and feedback. 

Managed by a community manager and an administrator with a monthly budget of $6,500 USD, the project aims to strengthen and grow the Akash ecosystem through active community participation and education. *Read “The Budget” Section below for a full breakdown of the costs of this campaign.

After 2 successful campaigns and 1 pilot, we believe we’ve gained enough insights to make this campaign more effective, cheaper and sustainable for the Akash community. 
The missions “User’s journey”

We designed a new “user journey”, a comprehensive path that guides new users through several key aspects of Akash Network:

- Introduce users to Akash: Start by educating them about what Akash is and its core concepts, with quizzes and polls to test participants knowledge
- Guide them through different tools like Akash Chat and Console
- Encourage active participation: Lead users towards contributing valuable PR’s on Github, starring repo’s and joining other Working groups
- Foster ongoing engagement: Motivate users to explore more advanced aspects of the Akash ecosystem by becoming insiders or participating in bounties
See missions offering for phase 3.0

![image](https://github.com/user-attachments/assets/fc88160f-0897-45c8-b828-ebae10248d8f)
![image](https://github.com/user-attachments/assets/1cbfa78a-ce28-4acb-9ebe-21bd977624d2)

- Welcome to Zealy Mission
- Join Akash socials
- Discover Akash Network: The Super Cloud
- Explore the Foundations and Evolution of Akash Network
- Explore Akash Chat: The Decentralized Alternative
- Star Akash Network GitHub repo’s
- Deploy on Akash using Akash Console
- WG and SIGs missions
- Join the Akash Insiders
- Bounties on Akash

This journey aims to transform newcomers into active, knowledgeable members of the Akash community. By structuring the onboarding process in this way, we can ensure that users gain a solid understanding of Akash, hands-on experience with the platform, and community initiatives.

Another benefit is that a lower number of missions will reduce our workload, margin for errors, system abuse and the cost of this campaign for the Akash community. 

### The budget 

For the next 3 months of campaigns we propose a $6,500 USD monthly budget, ($19,500 USD or 6,500 AKT for the length of the campaign). This is a 10% cut from previous campaigns and includes:
- 2 contributors pay - $1,500 USD each / month, total $3,000 USD / month
- Rewards - $3,000 USD / month
- Zealy subscription and other costs - $1,000 split between 3 months  

*We'll base all payments in USD for ease and clarity with a 10% buffer in case of volatility. 

### The team
**Robert Del Rey - InstaFinanzas:** 2 years old Akash insider, with vast experience leading projects. Will be responsible for managing missions, reviews, report issues, contact Zealy support when applicable, assists users on Discord and more tasks.
**Benji - Piber Dev:** 3 years old Akash insider, with experience as a tech support through the Vanguard Program and as an open-source developer. Will be reviewing technical missions, assisting with technical questions, and making sure everything is running as smoothly as possible during technical difficulties.

**As a reminder:**
The Akash Dev School Zealy campaign is being conducted for several important reasons:

- To incentivize direct contributions to Akash: The campaign aims to encourage developers and contributors to actively participate in the Akash network through structured missions and rewards.
- To enhance education: The project focuses on improving knowledge about Akash among participants.
- To increase engagement: A key goal is to boost engagement from developers and core contributors, which is measured through metrics like unique users, retention rate, and GitHub pull requests.
- To maintain community involvement: Regular communication and meetings are part of the campaign to keep the community engaged and involved in the project's progress.

### Conclusion

The Akash Developer Onboarding Program on Zealy has proven to be a successful initiative, driving engagement, education, and contributions within the Akash ecosystem. With the proposed Phase 3.0, we aim to build upon this success while optimizing for efficiency and sustainability.

**Key points of the new proposal include:**

- A streamlined "user journey" designed to guide participants from newcomers to active contributors
- A reduced budget to $6,500 USD per month, representing a 10% cut from previous campaigns
- A focused team of experienced Akash insiders to manage and support the program

By implementing these changes, we expect to maintain the program's effectiveness in onboarding new talent, fostering community growth, and driving meaningful contributions to the Akash network. This refined approach will ensure the long-term viability of the Developer Onboarding Program while continuing to serve as a vital component of Akash's community-driven development strategy.

We invite the Akash community to review this proposal and provide feedback as we work together to enhance and grow the Akash ecosystem through active participation and education.

**Comment by ghost** [comment]
We plan on discussing this for 1 week, then submit the prop on chain on Nov 25th

**Reply by Shunlexxi** [reply]
Let's go🚀

**Comment by andrewfowl** [comment]
Maybe it is a little bit off the topic but what if to engage people more, Akash would initiate a semi- or fully formal education program/certification about how to use Akash for deployment? I think it would help to increase retention once people have a good understanding of the system.

**Reply by ghost** [reply]
Appreciate your comment, I think you are referring to Akash EDU, this campaign focus on formal education on different aspects of Akash. It is on hold at the moment but is still actively discussed in the community

With Zealy we want to make it easy for newcomers to connect with Akash, by the end of the journey they should have a solid understanding of Akash and where to find opportunities to get involved and paid

**Reply by Shunlexxi** [reply]
yh, AkashEDU

**Reply by CaptainSplince** [reply]
Soon™️

**Comment by piber-dev** [comment]
Meeting notes (2024-11-18):
> In today's meeting, the recap and new proposal was presented by Robert in good detail with good reasoning, including showing examples of some of the new missions. The user journey was specifically explained in how it would get users to understand how to get involved on a deeper level by following a step-by-step set of missions. Core members Tyler Wright and Scott Hewitson joined the meeting and mentioned in chat that they appreciate the hard work put into this program onboarding new developers. Scott Hewitson and Benjamin (Piber) also discussed on how the core team could be helpful in turning Zealy users into active contributing developers.

**Comment by EzeFranklyn** [comment]
In "the budget" section, does that mean the whole rewards would now be in USD and not AKT?

**Reply by piber-dev** [reply]
thanks for your question @EzeFranklyn,

in previous campaigns and pilots we kept track of how much AKT we should allocate to each part of the proposal (role payments, prize pool, etc) when creating the proposal. this means that if we requested 10,000 AKT and half was supposed to go to the role payments, we paid out half to the role payments regardless of whether or not the AKT price changed a lot.

we realized now that this is not very sustainable for us personally as we expect a similar amount each month and AKT is a volatile asset so this needed to change. what we want to do now is when we pay out role payments, we will base the payments in USD when paying them instead of at the beginning of the proposal. this means that regardless of the price of AKT, we will make sure that the role payments are stable to the USD we are expecting each month. 

for example with the new payment plan we are going to follow: if 10,000 AKT were to out 50/50 to role payments and the rewards pool but AKT increased in price, either nothing changes or we can spend more on rewards. if AKT were to go down in price, we would have to work with the rewards pool budget a bit to make it stretch longer. in both cases, role payments are the same in USD.

promised rewards should not be affected by this and if volatility were to hit this proposal, the next proposal can be requested earlier to try make sure it does not affect our plans with how many sprints we want to run. rewards are also still likely going to be based in AKT as we don't want to promise e.g. $1000 worth of AKT to users in a scenario where AKT suddenly drops in price a lot.

**Reply by EzeFranklyn** [reply]
Okay, thanks for the clarification.

**Comment by CaptainSplince** [comment]
This is great gang. For reporting purposes Have you considered adding a funnel? 

Ex:
<img width="412" alt="funnel" src="https://github.com/user-attachments/assets/5e60362e-cb25-4e6a-a5a9-48f074aa37fe">

700 members join zealy 
400 join Akash Discord
100 post on Discord daily
10 apply and join the Insiders program.

Something like the above for each quest would be interesting to track.

Also, A nice addition would be some graphics showcasing Season 1 to 2 to 3 growth. 

Overall, really well done though.

**Reply by ghost** [reply]
Thanks for the suggestion top G!

Will definitely keep in mind 

About the graphics showcasing campaign growth, I quickly did this, let me know other stats worth adding 
https://docs.google.com/spreadsheets/d/1a9BRY6sg0Saip6Wz2gMnJwKKFhXISuYcoV8CHsye4Ns/edit?usp=sharing

**Comment by ghost** [comment]
Akash Zealy Phase 3.0 prop is on chain 

https://www.mintscan.io/akash/proposals/274

**Comment by Jiungnih** [comment]
Akash the best projects

**Comment by SophoclesSummer** [comment]
This proposal was approved on 12/2/2024 by a 90% majority. 
https://www.mintscan.io/akash/proposals/274
